src/Symfony/Component/ClassLoader/ClassCollectionLoader.php

```diff
@@ -216,6 +216,11 @@ public static function fixNamespaceDeclarations($source)
         $inNamespace = false;
         $tokens = token_get_all($source);
 
+        $nsTokens = [T_WHITESPACE => true, T_NS_SEPARATOR => true, T_STRING => true];
+        if (\defined('T_NAME_QUALIFIED')) {
+            $nsTokens[T_NAME_QUALIFIED] = true;
```

Comment:
This also misses `T_NAME_FULLY_QUALIFIED`

Reply:
Can you give an example that would result in `\T_NAME_FULLY_QUALIFIED` being emitted here?